mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -697,3 +698,66 @@ def __repr__(self):
         repr_str += '(bbox_enlarge_range={})'.format(
             self.bbox_enlarge_range.tolist())
         return repr_str
+
+
+@PIPELINES.register_module()
+class SweepPointSample(object):
+    """Sweep point voxel sample.
+
+    Apply voxel sampling to multiple sweep points.
+
+    Args:
+        name (str): Name of the dataset.
+        num_points (int): Number of points to be sampled.
+    """
+
+    def __init__(self, cur_sweep_cfg, prev_sweep_cfg):
+        self.cur_sweep_sampler = VoxelGenerator(**cur_sweep_cfg)
+        self.prev_sweep_sampler = VoxelGenerator(**prev_sweep_cfg)
```

Comment:
By default, we can set `prev_sweep_cfg=None` in case we do not need to sample sweep.